### ISSUE TITLE
Adding new feed causes all other articles to disappear - NC

### ISSUE THREAD

**Issue by deadwax** [readrops/Readrops#59]
Another article display issue under Nextcloud.  When I add a new feed in Readrops, all other articles, read and unread, disappear. There is no way to make them come back, refreshing does nothing. Articles published on other feeds after the new feed is added do start appearing as they are published.

NC 18.0.3
RD 1.1.3(9)

**Comment by Shinokuni** [comment]
> Another article display issue under Nextcloud. When I add a new feed in Readrops, all other articles, read and unread, disappear. There is no way to make them come back, refreshing does nothing

Did you encounter this problem when you have just started using the app or later ?

> Articles published on other feeds after the new feed is added do start appearing as they are published.

I'm sorry, I didn't understand what you mean by published. Are you talking about items you have synced after adding the feed ?

**Comment by deadwax** [comment]
This problem has come later, not at initial setup.

I mean as new articles appear in other feeds they appear in readrops, but articles that were visible before the new feed was added do not reappear. There are no articles visible that date to before the time I added the new feed.

**Comment by Shinokuni** [comment]
Do you remember if items disappeared when you just go back to the item list or after syncing ?

**Comment by deadwax** [comment]
The progression is this, which I have just verified with a different feed:

*Touch the plus button
*Copy in the feed URL
*Tick the feed and touch "validate"
*All articles disappear
*Pull to refresh - only articles from the new feed appear
*It is impossible to show any other articles from other feeds, except as new ones are published.

Deleting the new feed results in no articles at all being shown, with refreshing doing nothing.  It does not reverse the behaviour.

Verified with more than one feed.
